recipe/meta.yaml

```diff
@@ -12,34 +12,41 @@ source:
 
 build:
   number: 0
-  noarch: python
+  skip: True  # [py<35]
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation --ignore-installed --no-cache-dir -vvv
```

Comment:
are those --ignore-installed --no-cache-dir mandatory now ?

Reply:
No, they are optional

Reply:
Good thought. I'll modify the patch